sql/core/src/main/scala/org/apache/spark/sql/execution/Window.scala

```diff
@@ -920,7 +931,7 @@ private[execution] object AggregateProcessor {
     // Create the projections.
     val initialProjection = newMutableProjection(
       initialValues,
-      Seq(SizeBasedWindowFunction.n))()
+      Seq(partitionSize))()
```

Comment:
@hvanhovell  Why we only set a single column (partitionSize) as the input schema? I guess I am missing something...